src/mqtt/MQTT.cpp

```diff
@@ -241,7 +241,7 @@ void MQTT::onSend(const MeshPacket &mp, ChannelIndex chIndex)
         // handle json topic
         using namespace json11;
         auto jsonString = this->downstreamPacketToJson((MeshPacket *)&mp);
-        if (!jsonString.isEmpty()) {
+        if (jsonString.length() != 0) {
```

Comment:
Could this potentially have a null string?

Reply:
I don't think so. When I read the doc correctly, then `length()` does not count the zero termination. I changed this from `isEmpty()` because some of the Arduino targets use a different `String` implementation that do not have `isEmpty()`. 

To be sure, I also checked the Arduino impl of `length()` and it seems to be consistent with the `std::string` impl:

```
Returns the length of the String, in characters. (Note that this doesn’t include a trailing null character.)
```